arch/xtensa/src/esp32s2/Make.defs

```diff
@@ -52,6 +52,10 @@ ifeq ($(CONFIG_DEBUG_ALERT),y)
   CMN_CSRCS += xtensa_dumpstate.c
 endif
 
+ifeq ($(CONFIG_ARCH_FPU),y)
```

Comment:
We can remove this one, ESP32-S2 does not have an FPU.

Reply:
Done.